forge-gui/res/cardsfolder/upcoming/marching_duodrone.txt

```diff
@@ -0,0 +1,7 @@
+Name:Marching Duodrone
+ManaCost:2
+Types:Artifact Creature Construct
+PT:2/2
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ DBTreasure | TriggerDescription$ Whenever CARDNAME attacks, each player creates a Treasure token.
```

Comment:
TriggerDescription$ Whenever CARDNAME attacks, each player creates a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")